scripts/generate-patch

```diff
@@ -20,9 +20,14 @@ function revert_crd_changes {
 		echo "No directory provided to revert charts-crd changes within"
 		exit 1
 	fi
-	# Move charts-crd/templates back into charts/crd/
+	# Move CRDs back into charts/crd/
 	mkdir -p ${f}/charts/crds/
-	mv ${f}/charts-crd/templates/* ${f}/charts/crds/
+	if [[ -d ${f}/charts-crd/crd-manifest ]]; then
+		mv ${f}/charts-crd/crd-manifest/* ${f}/charts/crds/
+	else
+		mv ${f}/charts-crd/templates/* ${f}/charts/crds/
+	fi
+
```

Comment:
Based on discussion in https://github.com/rancher/charts/pull/607#discussion_r483288658, the work to split this into a `clean-crds` script that is called on clean and before patch is applied should be done as part of this PR.

Will update in the next commit.

Reply:
We can follow up on this now, as that is merged

Reply:
Updated in latest push